examples/data-transfer-pg/main.py

```diff
@@ -62,41 +69,62 @@ def connect_to_databases():
             os.environ['DB2_USERNAME'], 
             os.environ['DB2_PASSWORD']
         )
+
     return source_db_conn, dest_db_conn
 
 
+def get_data(count: int) -> List[Tuple]:
+    ''' Generate user data. '''
+    fake = Faker()
+    row = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
+
+    return [row(i) for i in range(count)]
+
+
+def insert_source_db(data: List):
+    ''' Insert query for generating data on source table'''
+    for i in data: 
+        with open('sql/source_db_setup.sql', 'a') as file:
+            file.write(SQL_INSERT_DATA.format(
+            id = i[0], 
+            name = i[1], 
+            city = i[2]
+        )+ '\n')
```

Comment:
@yankeexe You don't need `open` to read from files. Use the existing function we created `extract_sql` - https://github.com/laudio/pyodbc/pull/29/files#diff-e21edbaba8bb446370433c6b99efbb20R95